spec/rubocop/cop/rails/date_and_time_column_name_spec.rb

```diff
@@ -0,0 +1,136 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::DateAndTimeColumnName do
+  subject(:cop) { described_class.new }
+
+  context 'when `add_column` method' do
+    described_class::TYPE_TO_SUFFIX.each do |type, suffix|
```

Comment:
This won't fail if you remove or add entries to `TYPE_TO_SUFFIX`. It also requires the reader of the spec to look it up in the implementation.

Reply:
That is the way how similar things are tested in many places in the `rubocop` and `rubocop-rails` (for example, https://github.com/rubocop/rubocop-rails/blob/647e46a407cb11300d2525e6760abf5b73cc440b/spec/rubocop/cop/rails/validation_spec.rb#L28). I was just stayed consistent with that style.